rust/extractor/src/main.rs

```diff
@@ -118,3 +125,19 @@ fn main() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+fn main() -> anyhow::Result<()> {
+    let cfg = config::Config::extract().context("failed to load configuration")?;
+    let qltest = cfg.qltest;
+    let qltest_log = cfg.log_dir.join("qltest.log");
+    let result = std::panic::catch_unwind(|| run_extractor(cfg));
```

Comment:
Are you sure panic catching is desired? Did you encounter any unavoidable `panic`s?

Reply:
Ok, that makes sense. I was worried you had this to catch some know panic causes. In general we should avoid unwraps as much as possible (unless we're really sure). Things like `expect` with a sensible error string are much easier for debugging.